tests/test_D1_model_components.py

```diff
@@ -0,0 +1,177 @@
+import pytest
+import pandas as pd
+import os
+import json
+import oemof.solph as solph
+
+# internal imports
+import src.D1_model_components as D1
+from tests.constants import (
+    D1_JSON
+)
+
+
+@pytest.fixture()
+def get_json():
```

Comment:
an inline comment would help test beginners to understand why the `@pytest.fixture()` here